Dockerfile

```diff
@@ -2,7 +2,7 @@ FROM mcr.microsoft.com/dotnet/aspnet:7.0 AS base
 WORKDIR /app
 EXPOSE 80
 
-FROM mcr.microsoft.com/dotnet/sdk:7.0.304 AS build
+FROM mcr.microsoft.com/dotnet/sdk:7.0.400 AS build
```

Comment:
if the whole variables thing I said in the other comment works out, you can make this an `ARG` and pass it from the yaml.